client/components/theme/index.jsx

```diff
@@ -172,8 +176,12 @@ const Theme = React.createClass( {
 								context: 'singular noun, the currently active theme'
 							} ) }</span>
 						}
-						{ price && ! purchased &&
-							<span className="theme-badge__price">{ price }</span>
+						{
+							config.isEnabled( 'jetpack/pijp' )
+							? ! this.props.isJetpack && price && ! purchased &&
```

Comment:
Aside, what is this doing? Hiding the price of a premium theme on JP sites if the feature flag isn't active doesn't seem to make much sense to me -- aren't premium themes going to be hidden altogether in that case?